Makefile

```diff
@@ -57,6 +57,14 @@ BABEL := $(NODE_LOCAL_BIN)/babel
 ESLINT := $(NODE_LOCAL_BIN)/eslint
 FLAKE8 := $(NODE_LOCAL_BIN)/flake8
 
+ifdef DEV_XPI
+LOOP_XPI_DATE=`date +"%Y%m%d%H%M"`
+LOOP_DEV_XPI=-D LOOP_DEV_XPI=1
```

Comment:
I'd suggest calling this LOOP_DEV_XPI_DEFS or something.  Right now, reading the call sites is a bit confusing, as it looks like there might be an XPI file in the middle of the commands.